src/main/java/com/recurse/portfolio/web/ProjectController.java

```diff
@@ -50,7 +52,7 @@ public ModelAndView getNewProject(
             .addObject("allTags", tagRepository.findAll());
     }
 
-    @PostMapping("/project/")
+    @PostMapping("/project/new")
```

Comment:
I almost did this initially, and probably I should have. Tell me more about your motivation for changing it? Was it for the sake of clarity?

Reply:
We noticed the GetMapping and PostMapping URIs for new projects didn't match, so we added _/new_ to the PostMapping URI to keep things consistent.

Reply:
Makes sense, thank you!